src/test/groovy/org/gradle/profiler/ProfilerIntegrationTest.groovy

```diff
@@ -1057,10 +1057,10 @@ println "User home: \$gradle.gradleUserHomeDir"
 
         when:
         new Main().run("--project-dir", projectDir.absolutePath, "--output-dir", outputDir.absolutePath, "--gradle-version", minimalSupportedGradleVersion,
-            "--benchmark", "--gradle-user-home", "foobar", "help")
+            "--benchmark", "--gradle-user-home", "home with spaces", "help")
```

Comment:
The test is not specifying a Gradle home but a Gradle _user_ home. IIUC, that does not cause problems. I didn't find any test which uses a not auto-installed Gradle version :(

Reply:
Sorry, I meant Gradle user home, and it does cause a problem (I checked). I think we're good.